draft-ietf-rats-architecture.md

```diff
@@ -1317,3 +1317,160 @@ use beyond the period for which it deems the Attestation Result to remain
 valid.  Thus, if the Attestation Result sends a validity lifetime
 in terms of time(RX)-time(RG), then the Relying Party can check
 `time(OP) - time(ER) < time(RX)-time(RG)`.
+
+# Appendix B: System Examples
+
+(Rough first draft to see if there is interest)
+
+## System Example with OS Vendor as Main Driver
+
+Three entities (companies / organizations) participate in this examples.
+
+The operating system vendor is at the center. It sets the architecture and the level of security achieved. It performs the following:
+* Generates large quantities of ECDSA key pairs to be used as per-device attestation keys
+* Signs the public keys using an attestation PKI and root that they control creating per-device certificates.
+* Securely sends the private keys and certificates to the device OEM in batches to be used in manufacturing
+* Authors the device SW that is the Attester and ships it to the OEMs. This is a part of the OS.
+* Publishes the root certificate and definition of the claims
+
+The device OEM does the manufacturing. It performs the following:
+* Physically makes the device to OS vendors security requirements
+* Installs the OS on the device
+* Personalizes each device with a unique private key and certificate supplied by the OS vendor
+
+The application vendor that implements some application with a back-end cloud service. It performs the following:
+* Author an app on the device that calls the attestation API to obtain Attestation Evidence
+* Convey the Attestation Evidence to the back-end cloud service over the network
+* Verify the Attestation Evidence and decided whether or not to allow application service
+
+This maps as follows in the RATS architecture:
+
+Endorser:
+
+: The operating system vendor is the endorser
+
+Endorsement:
+
+: The root certificate published by the endorser
```

Comment:
I can see it either way. I picked the root because the root flows directly from the Endorser to the Verifier, where as the device's cert does not flow directly.